common/app/routes/challenges/components/Bug-Modal.jsx

```diff
@@ -25,12 +25,11 @@ export class BugModal extends PureComponent {
     } = this.props;
     return (
       <Modal
-        onHide={ closeBugModal }
         show={ isOpen }
         >
         <Modal.Header className='challenge-list-header'>
           Did you find a bug?
-          <span className='close closing-x'>×</span>
+          <span className='close closing-x' onClick={ closeBugModal }>×</span>
```

Comment:
Multi-attributes must be spread across lines. I'm surprised the linter didn't throw an error.